code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -15,9 +15,11 @@
 	container_type = OPENCONTAINER
 	slot_flags = SLOT_FLAG_BELT
 	/// If TRUE, the hypospray can inject through most hardsuits/modsuits.
-	var/can_pierce_hardsuits = FALSE
+	var/penetrate_thick = FALSE
 	/// If TRUE, the hypospray isn't blocked by suits with TRAIT_HYPOSPRAY_IMMUNE.
 	var/ignore_hypospray_immunity = FALSE
+	/// if TRUE, the hypospray will always succeed at injecting an organic limb regardless of protective clothing or traits (except for TRAIT_HYPOSPRAY_IMMUNE).
+	var/defeat_all_protection = FALSE
```

Comment:
`defeat_all_protection` is probably not the best var name, something like `pierce_all` or `penetrate_all` is shorter, and slightly more self descriptive

Reply:
Changed.